### ISSUE TITLE
build fails

### ISSUE THREAD

**Issue by akoerner1** [DLR-TS/adore_v2x_sim#1]
## Steps to reproduce
1. clone the repo
2. run make
```bash
make
...
#20 2.509 [  2%] Built target cpm_v1_19_cpm_pdu_descriptions_generate_messages_cpp
#20 2.514 /tmp/adore_v2x_sim/src/channel_sim_node.cpp:20:10: fatal error: adore_v2x_sim/SimSPATEM.h: No such file or directory
#20 2.514    20 | #include <adore_v2x_sim/SimSPATEM.h>
#20 2.514       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
#20 2.514 compilation terminated.
#20 2.518 Scanning dependencies of target mcm_dmove_mcm_dmove_generate_messages_cpp
#20 2.521 make[2]: *** [CMakeFiles/channel_sim_node.dir/build.make:63: CMakeFiles/channel_sim_node.dir/src/channel_sim_node.cpp.o] Error 1
#20 2.521 make[1]: *** [CMakeFiles/Makefile2:264: CMakeFiles/channel_sim_node.dir/all] Error 2
#20 2.521 make[1]: *** Waiting for unfinished jobs....
#20 2.525 [  2%] Built target mcm_dmove_mcm_dmove_generate_messages_cpp
#20 2.525 make: *** [Makefile:163: all] Error 2
```

**Comment by daniel-gitgit** [comment]
Sorry, can't reproduce! I did the following:
1. clone
2. git submodule update --init --recursive
3. make (no errors)
4. make clean (maybe there is an old image, which is not re-built?) 
5. make (no errors again)

Some observations:
- the header file should be generated by the ros msg compilation process
- We need a jenkins job for each adore submodule, also this one!

**Comment by akoerner1** [comment]
This issue is resolved by with commit: [554436e3f78a2a81711da7b14de252b1b5a3c860](https://github.com/DLR-TS/adore_v2x_sim/commit/554436e3f78a2a81711da7b14de252b1b5a3c860)

The failure is reproducable in a newly provisioned virtual machine with ubuntu 22.04. 

The root cause is not fully understood without deeper investigation however the target defined in adore_v2x_sim "channel_sim_node" was missing "add_dependencies" and "target_include_directories" which is required by any target consuming autogenerated headers.

Running the following commands should result in a successful build of adore_v2x_sim after an update:
```bash
cd adore_v2x_sim
git pull
git checkout 554436e3f78a2a81711da7b14de252b1b5a3c860
make
```
